### ISSUE TITLE
.getPersonalAccessToken() throws a 404

### ISSUE THREAD

**Issue by robertvpaf** [contentful/contentful-management.js#223]
Calling `.getPersonalAccessToken(xxxx)` always throws a HTTP 404 error. 
```
const {createClient} = require('contentful-management');

const client = createClient({
	accessToken: 'xxxxx'
});

client.getPersonalAccessToken('xxxxxx').then(console.log);
```
The issue seems to be that the code is doing a HTTP POST request while the reference documentation says to use a GET request to `/users/me/access_tokens/{token_id}`.

**Comment by robertvpaf** [comment]
Pull request with a fix: https://github.com/contentful/contentful-management.js/pull/224

**Comment by robertvpaf** [comment]
Fixed in version 5.9.1
